src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,54 @@
+import { Dispatch, SetStateAction } from 'react';
+
+import cn from 'classnames';
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  filterStatus: FilterStatus;
+  setFilterStatus: Dispatch<SetStateAction<FilterStatus>>;
```

Comment:
u can use another syntax(up to u)
```suggestion
    setFilterStatus: (status: FilterStatus) => void;
```